frontend/public/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta name="twitter:domain" content="ethwallpaper.co" />
+  <meta name="twitter:card" content="summary" />
```

Comment:
just tested this.  in your next commit, can we change this to `summary_large_image` instead of `summary`? 

id PR it but it seems like it'd be easier to just change this way :()

Reply:
done

Reply:
weird, still shows as `summary` here, and on ethwallpaper.co